CMakeLists.txt

```diff
@@ -31,14 +31,21 @@ include(GNUInstallDirs)
 if(BEMAN_EXEMPLAR_BUILD_TESTS)
     # Fetch GoogleTest
     FetchContent_Declare(
-        GTest
+        googletest
+        FETCHCONTENT_TRY_FIND_PACKAGE_MODE
+        ALWAYS
         GIT_REPOSITORY https://github.com/google/googletest.git
-        GIT_TAG
-            6910c9d9165801d8827d628cb72eb7ea9dd538c5 # release-1.16.0
+        GIT_TAG v1.16.0
+        FIND_PACKAGE_ARGS 1.16.0 NAMES GTest COMPONENTS gmain_main
```

Comment:
What is this line for? @ClausKlein 

I couldn't find any corresponding documentation about this flag https://cmake.org/cmake/help/latest/command/find_package.html

Reply:
[FIND_PACKAGE_ARGS](https://cmake.org/cmake/help/latest/module/FetchContent.html#command:fetchcontent_declare)